rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -319,7 +336,7 @@ Use this credential for accessing the Microsoft Outlook email application via Mi
 The host address used to get the credentials. The default address for Outlook services is `https://login.microsoftonline.com/common/oauth2/v2.0/`. This address might change if the customer uses a self hosted service.
 *** *Client ID*:
 +
-To authenticate with the OAuth Host, an Azure application must be defined in the customer's space. This Azure application has an ID that is unique in the entire OAuth Host space and that allows users to explicitly grant or revoke access for this Azure application to their accounts.
+To authenticate with the OAuth Host, an Azure application must be defined in the customer's space. This Azure application has an ID, which is unique in the entire OAuth Host space, and that allows users to explicitly grant or revoke access for this Azure application to their accounts.
```

Comment:
```suggestion
To authenticate with the OAuth Host, an Azure application must be defined in the customer's space. This Azure application must have an ID that is unique in the entire OAuth Host space and that allows users to explicitly grant or revoke access for this Azure application to their accounts.
```

Reply:
Every Azure application has a unique ID, I suppose. This text was provided by our developers. Unfortunately I don't have an Azure account, so I can't test this.